CHANGES.md

```diff
@@ -1,5 +1,25 @@
 # Change log for kotlinx.coroutines
 
+## Version 0.26.0
+* Major rework of `kotlinx.coroutines` concurrency model (see #410 for a full explanation of the rationale behind this change):
+  * All coroutine builders are now extensions on `CoroutineScope` and inherit its `coroutineContext`. Standalone builders are deprecated for removal.
+  * As a consequence, all nested coroutines launched via builders are now automatically establish parent-child relationship and inherit `CoroutineDispatcher`.
+  * All coroutine builders use `Dispatchers.Default` by default if `CoroutineInterceptor` is not present in their context.
+  * [CoroutineScope](https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines.experimental/-coroutine-scope/) became the first-class citizen in `kolinx.coroutines`.
+  * `withContext` `block` argument has `CoroutineScope` as a receiver.
+  * [GlobalScope](https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines.experimental/-global-scope/) is introduced to simplify migration to new API and to launch global-level coroutines.
+  * `currentScope` and `coroutineScope` builders are introduced to extract and provide `CoroutineScope`.
+  * Factory methods to create `CoroutineScope` from `CoroutineContext` are introduced.
+  * `CoroutineScope.isActive` became an extension property.
+  * New sections about structured concurrency in core and UI guides.
```

Comment:
Links to the corresponding sections are nice to have here, so that people can click to quickly read them.